conf/openshift/openshift.json

```diff
@@ -162,7 +163,7 @@
         "name": "iqss-dataverse-solr"
       },
       "spec": {
-        "dockerImageRepository": "iqss/dataverse-solr"
+        "dockerImageRepository": "iqss/dataverse-solr:4762"
```

Comment:
huh?

Reply:
This is an unintentional change that came from a commit I cherry-picked. Removed.